src/components/SingleEvent/Admin/CheckIn.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react'
+import styled from 'react-emotion'
+
+import Button from '../../Forms/Button'
+import TextInput from '../../Forms/TextInput'
+import Label from '../../Forms/Label'
+
+import { withApollo } from 'react-apollo'
+import { POAP_USERS_QUERY } from '../../../graphql/queries'
+import { MARK_USER_ATTENDED } from '../../../graphql/mutations'
+
+const Section = styled('section')`
+  margin-bottom: 40px;
+`
+
+const EventIdInputContainer = styled('div')`
+  display: flex;
+  margin-bottom: 20px;
+`
+
+const EventIdInput = styled(TextInput)`
+  margin: 0;
+  margin-right: 20px;
+`
+
+export default withApollo(function CheckIn({ party, client }) {
+  const [poapId, setPoapId] = useState('')
+
+  const checkIn = async () => {
+    const {
+      data: { addresses }
+    } = await client.query({
+      query: POAP_USERS_QUERY,
+      variables: { eventId: poapId }
+    })
+
+    const attendees = party.participants.filter(participant =>
+      addresses.includes(participant.user.address)
+    )
+
+    attendees.map(participant =>
```

Comment:
`map` doesn't return into a variable here, so let's just turn this into a `forEach` to be more explicit